src/locale/locales/fr/messages.po

```diff
@@ -6415,6 +6456,10 @@ msgstr "Nous sommes désolés ! Vous ne pouvez vous abonner qu’à vingt étiq
 msgid "Welcome back!"
 msgstr "Bienvenue !"
 
+#: src/components/NewskieDialog.tsx:103
+msgid "Welcome, friend!"
+msgstr "Bienvenue, mon ami !"
```

Comment:
`mon ami`, or `mon amie`? yeah, complicated.

We can workaround the problem by acknowledging the context: it's the string shown when we show an infobox about ourselves and we are a newsky. So any expression we use in that context should work.

```suggestion
msgstr "Bienvenue et enchanté !"
```

Reply:
D'oh, I completely forgot that _écriture inclusive_ would apply here, of course.

But you suggested a great workaround 👍